orchestrationSpecs/packages/schemas/src/zodMeta.ts

```diff
@@ -0,0 +1,180 @@
+/**
+ * Zod GlobalMeta Type Extension
+ * 
+ * This module extends Zod's GlobalMeta interface to define UI metadata
+ * that flows through to OpenAPI/JSON Schema via @asteasolutions/zod-to-openapi.
+ * 
+ * Usage:
+ * ```typescript
+ * import 'zod';
+ * import './zodMeta'; // Side-effect import to extend GlobalMeta
+ * 
+ * const schema = z.string().meta({
+ *   title: "Field Title",
+ *   placeholder: "Enter value...",
+ *   group: "my-group",
+ *   order: 1
+ * });
+ * ```
+ */
+
+import 'zod';
+
+declare module 'zod' {
+    interface GlobalMeta {
+        // ============================================
+        // UI Presentation
+        // ============================================
+        
+        /** Display title for the field (overrides auto-generated label) */
+        title?: string;
+        
+        /** Placeholder text for input fields */
+        placeholder?: string;
+        
+        /** Constraint text shown below the field (e.g., "Must be at least 8 characters") */
+        constraintText?: string;
+        
+        /** Help text shown in a tooltip or info icon */
+        helpText?: string;
+        
+        // ============================================
+        // Field Configuration
+        // ============================================
+        
+        /** 
+         * Explicit field type override.
+         * If not specified, the type is inferred from the Zod schema.
+         */
+        fieldType?: 
```

Comment:
This one is obviously not a concern for anything but a webpage.  I'd recommend renaming some of these and collapsing others, then having something that maps those values to your UI values (like toggle vs checkbox).